### ISSUE TITLE
[Feature Request] Support for Season Title

### ISSUE THREAD

**Issue by roanv** [joshuaavalon/AvalonXmlAgent.bundle#4]
Absolutely love this agent!

One thing I would like to do is have season titles also be updated via XML. 

I am not sure how possible this is via agent since Plex does not support editing the season title through the interface.

It does however store and display a season title, and you can edit it via javascript.

https://forums.plex.tv/t/custom-names-for-seasons/204183

Currently I'm using this to manually rename my seasons but it gets quite tedious. I would love to just do it via XML and not have to worry about losing it when I eventually move servers again.

Cheers

**Comment by joshuaavalon** [comment]
No, it is not possible. Plex does not support season name. The JavaScript just change how it display in your browser only. It does not actually do any thing on the server.

**Comment by roanvonb** [comment]
Hi Joshua, thanks for getting back to me!

It actually [does](https://www.reddit.com/r/PleX/comments/9guvvr/tool_for_renaming_tv_seasons/e66yftn?utm_source=share&utm_medium=web2x&context=3
) save it to the server rather than being a local browser change. I have confirmed this on my own server as well. It applies and persists for all my users on all platforms. 

From what I can tell it works by injecting an input element called "title" when editing a season. You run the script while you have the "[edit season](https://i.imgur.com/f651F20.jpg)" box open and it adds a field called "NAME" to the dialogue box.

Here is the actual JS:

```
javascript: var ta = document.getElementById("lockable-summary");

if (!ta) 
{
    alert("Can't find submission form!");
    exit;
}
var outer = document.createElement("div");
var nameLabel = document.createElement("div");
var nameOb = document.createElement("input");
var attName = document.createAttribute("name");
attName.value = "title";
nameOb.setAttributeNode(attName);
nameLabel.innerHTML = "NAME:";
outer.appendChild(nameLabel);
outer.appendChild(nameOb);
ta.parentNode.appendChild(outer);
exit;
```
There is also (natively) the option to add a summary to each season, which I would love to be able to do via XML as well.

Cheers,
Roan
